src/cmdline.rs

```diff
@@ -54,6 +54,11 @@ pub fn create_expected_args<'a>() -> App<'a, 'a> {
                      .help("Adds single features")
                      .multiple(true)
                      .takes_value(true))
+                  .arg(Arg::with_name("build_cache_without_training")
```

Comment:
Maybe short name could be `build_cache_only`, not to be the same as the long one?

Reply:
done